classes/core/PKPRequest.inc.php

```diff
@@ -581,6 +591,18 @@ function &getUser() {
 	function getUserVar($key) {
 		$_this = PKPRequest::_checkThis();
 
+		if (is_a($_this, 'Request')) {
```

Comment:
@kaschioudi, I think we already worked through this (or another similar case) -- OTOH `$_this` will _always_ be a `Request`. My apologies if I forgot, which also happens.